### ISSUE TITLE
New / editing transcription lines missing focus outline in Safari

### ISSUE THREAD

**Issue by srallen** [zooniverse/front-end-monorepo#2177]
## Describe the bug

First reported in #2159 :

![](https://user-images.githubusercontent.com/23665803/117719503-17ae0c00-b1a3-11eb-9c05-ca2464326656.jpg)

## Expected behavior

Outline should appear in Safari when focused.
